src/main/java/com/reckue/post/transfers/nodes/image/ImageNodeResponse.java

```diff
@@ -9,6 +9,7 @@
  *
  * @author Kamila Meshcheryakova
  */
+@EqualsAndHashCode(callSuper = true)
```

Comment:
I want to ask, why did you add "true" not "false"?

Reply:
Because we override equals and hashcode for the superclass field as well

Reply:
Because we override equals and hashcode for the superclass field as well